qiskit/providers/ibmq/api/clients/account.py

```diff
@@ -169,46 +173,54 @@ def job_submit(
 
         if not submit_info:
             # Submit Qobj via HTTP.
-            submit_info = self._job_submit_post(backend_name, qobj_dict, job_name)
+            submit_info = self._job_submit_post(backend_name, qobj_dict, job_name, job_share_level)
 
         return submit_info
 
     def _job_submit_post(
             self,
             backend_name: str,
             qobj_dict: Dict[str, Any],
-            job_name: Optional[str] = None
+            job_name: Optional[str] = None,
+            job_share_level: Optional[ApiJobShareLevel] = None
     ) -> Dict[str, Any]:
         """Submit a Qobj to a device using HTTP POST.
 
         Args:
             backend_name: the name of the backend.
             qobj_dict: the Qobj to be executed, as a dictionary.
             job_name: custom name to be assigned to the job.
+            job_share_level: level the job should be shared at.
 
         Returns:
             job status.
         """
-        return self.client_api.job_submit(backend_name, qobj_dict, job_name)
+        return self.client_api.job_submit(
+            backend_name, qobj_dict, job_name, job_share_level=job_share_level.value)
 
     def _job_submit_object_storage(
             self,
             backend_name: str,
             qobj_dict: Dict[str, Any],
-            job_name: Optional[str] = None
+            job_name: Optional[str] = None,
+            job_share_level: Optional[ApiJobShareLevel] = None
     ) -> Dict:
         """Submit a Qobj to a device using object storage.
 
         Args:
             backend_name: the name of the backend.
             qobj_dict: the Qobj to be executed, as a dictionary.
             job_name: custom name to be assigned to the job.
+            job_share_level: level the job should be shared at.
 
         Returns:
             job status.
         """
         # Get the job via object storage.
-        job_info = self.client_api.submit_job_object_storage(backend_name, job_name=job_name)
+        job_info = self.client_api.submit_job_object_storage(
+            backend_name,
+            job_name=job_name,
+            job_share_level=job_share_level.value)
```

Comment:
If `job_share_level` is really optional (as per 206), this line might fail?

Reply:
Yeah, that would be the case. Thanks for the catch! Went ahead and updated the two places where this would be an issue.